### DISCUSSION TITLE
A error in my code

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1076] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91677627/154221216-90e4a0d4-47df-43e7-96fd-430899f12530.png)

**Comment by Noveleader** [comment]
I was doing a different mistake name of my smart contract is SimpleStorage
Would be better if anyone tell the me little deep about line 7

**Comment by cromewar** [comment, marked as answer]
Hello @Noveleader 
Please share the following:
1. another screenshot of the project folder structure (want to see the name of the sol files).
2. A screenshot of you SimpleStorage contract.

Important note, remember to use Camel Case for contract names, they should start with Upper case letter and any consequent word also should be upper case, so instead of:

`Simplestorage`

this should be:

`SimpleStorage`

**Reply by Noveleader** [reply]
Yeah I would remember that it is like class as in other languages, I was calling the name of my contract wrong.

Can you help out with the syntax of line 7 what it actually refers.
Thanks

**Reply by cromewar** [reply]
Well based on your code:

Simplestorage simplestorage = new Simplestorage();

Simplestorage is the data type, you can think about it as a class on OOP, you are saying that the variable type is going to be Simplestorage, like saying the variable is going to be uint256 or so.
simplestorage is the name of the variable, this could be named "contract", "bryan" or whatever you want to, is does not matter but is a good practice to use descriptive names.
new Simplestorage() : Creates a new instance of the contract you are importing from.

**Reply by Noveleader** [reply]
Got it thanks 👍

**Reply by Noveleader** [reply]
So since Simplestorage is my contract name I can specify it like this only right?

**Reply by cromewar** [reply]
Ye sir!

**Reply by Noveleader** [reply]
Alright! This community is lovely, I haven't seen any community like this before.
Thanks
